<?xml version="1.0"?>
<launch>
  <arg name="config_file" default="$(find lvio_fusion_node)/config/kitti.yaml" />

  <node name="lvio_fusion_node" pkg="lvio_fusion_node" type="lvio_fusion_node" output="screen">
    <param name="config_file" value="$(arg config_file)" />
  </node>

  <node name="rl_fusion" pkg="rl_fusion" type="rl_fusion_node" output="screen">
    <param name="height" value="4" />
    <param name="width" value="12" />
  </node>

  <!-- <node name="rviz" pkg="rviz" type="rviz" output="log" args="-d $(find lvio_fusion_node)/config/default.rviz"/> -->
  
</launch>
